.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+php:
+  - 7.4
+
+before_script:
+  - composer install --dev --no-interaction
+
+script:
+  - php vendor/bin/phpunit -c phpunit.xml
+
+after_success:
+  - travis_retry php vendor/bin/php-coveralls -v
```

Comment:
the `travis_retry` is necessary? I believe the code run without `travis_retry`, only putting `php vendor/bin/php-coveralls --verbose`.

Reply:
I don't have much knowledge in Travis too, and I just followed the instructions on the README from coveralls repo.